File: app/src/main/java/com/skeleton/mvp/ui/onboarding/signin/SignInView.java
package com.skeleton.mvp.ui.onboarding.signin;


import com.skeleton.mvp.ui.base.BaseView;

/**
 * Developer: Click Labs
 */

public interface SignInView extends BaseView {

    /**
     * On sign in success.
     *
     * @param message the message
     */
    void onSignInSuccess(String message);

    /**
     * Show phone number error.
     *
     * @param resId the res id
     */
    void showPhoneNumberError(int resId);
}

File: app/src/main/java/com/skeleton/mvp/ui/onboarding/signin/SignInView.kt
package com.skeleton.mvp.ui.onboarding.signin;

import com.skeleton.mvp.ui.base.BaseView


/**
 * Developer: Click Labs
 */
interface SignInView : BaseView {

    /**
     * On sign in success.
     *
     * @param message the message
     */
    fun onSignInSuccess(message: String)

    /**
     * Show phone number error.
     *
     * @param resId the res id
     */
    fun showPhoneNumberError(resId: Int)
}
